compile/inc/src/main/scala/sbt/inc/APIs.scala

```diff
@@ -38,6 +38,8 @@ trait APIs {
 
   def internal: Map[File, Source]
   def external: Map[String, Source]
+
+  def hasPackageObject(src: File): Boolean
```

Comment:
Oh, I see. In the past we've been adding new methods without considering it as a binary compatibility breakage. I could maybe make this method `private[sbt]`. I'm not sure what would be best.

In any case, this shouldn't break existing plugins.